Error: '"@angular/core"' has no exported member named 'OutputRef'

Hi,
within my angular project using
`"@angular/core": "17.1.0"`
and devDependencies set to
```
    "@testing-library/angular": "17.1.0",
    "@testing-library/jasmine-dom": "^1.2.0",
    "@types/jasmine": "^5.1.4",
    "cheerio": "^1.0.0-rc.12",
    "jasmine-core": "^5.1.2",
```
I get the error

> error TS2724: '"@angular/core"' has no exported member named 'OutputRef'. Did you mean 'Output'?

![Bildschirmfoto 2024-08-07 um 13 09 44](https://github.com/user-attachments/assets/72065024-bd1a-4169-9785-30492b1dc140)

on my karma and also wallabyjs server.

This resolves itself when downgrading back to `"@testing-library/angular": "17.0.0",`

--- a/projects/testing-library/src/lib/models.ts
+++ b/projects/testing-library/src/lib/models.ts
@@ -1,7 +1,15 @@
-import { Type, DebugElement, OutputRef, EventEmitter, Signal } from '@angular/core';
+import { Type, DebugElement, EventEmitter, Signal } from '@angular/core';
 import { ComponentFixture, DeferBlockBehavior, DeferBlockState, TestBed } from '@angular/core/testing';
 import { Routes } from '@angular/router';
 import { BoundFunction, Queries, queries, Config as dtlConfig, PrettyDOMOptions } from '@testing-library/dom';
+
+// TODO: import from Angular (is a breaking change)
+interface OutputRef<T> {
+  subscribe(callback: (value: T) => void): OutputRefSubscription;
+}
+interface OutputRefSubscription {
+  unsubscribe(): void;
+}
 
 export type OutputRefKeysWithCallback<T> = {
   [key in keyof T]?: T[key] extends EventEmitter<infer U>

